scripts/sign_up.py

```diff
@@ -53,11 +86,11 @@ async def main() -> int:
                 RETURNING account_id
             """,
             values={
-                "username": input("Username: "),
-                "email_address": input("Email address: "),
-                "privileges": 2_147_483_647,
-                "password": hash_password(getpass("Password: ")).decode(),
-                "country": input("Country: "),
+                "username": username,
+                "email_address": email_address,
+                "privileges": privileges,
+                "password": password,
+                "country": country,
```

Comment:
we should align on whether these should be upper or lower case in db perhaps

Reply:
for now, let's go with upper
```suggestion
                "country": country.upper(),
```